app/assets/json/translations/locale-es.json

```diff
@@ -647,6 +646,7 @@
             "preference_proof_address_documents": "¿Qué tipo de documento desea cargar para demostrar que vive en esa dirección?",
             "preference_proof_documents": "¿Qué documento va a cargar para demostrar su elegibilidad?",
             "preferences": "Preferencias",
+            "primary_language": {},
```

Comment:
not sure why these pop up and not sure how to stop them from popping up when I run grunt translations. They cause issues because then if I run it again it fails